app/services/redact_service.rb

```diff
@@ -8,8 +8,20 @@ def call
       end
     end
 
+    def redact_old_client_refs
+      RequestLog.find_each.select { |log| log.created_at < 14.days.ago }.each do |li|
```

Comment:
@starswan What do you think of using `RequestLog.where("created_at < ?" , 14.days.ago)`

<img width="1608" alt="Screenshot 2023-08-04 at 12 12 57" src="https://github.com/ministryofjustice/cfe-civil/assets/195928/2e3519ef-2e38-404f-afae-0b1175fb500b">

Reply:
What I have written is more idiomatic for a batch job. If there were *lots* then the query would generate a large dataset in memory - obviously it probably doesn't matter here :-)

Reply:
@starswan What about `RequestLog.where("created_at < ?" , 14.days.ago).find_each do` ?